test/e2e/connectivity_test.go

```diff
@@ -104,7 +104,7 @@ func createPodsOnDifferentNodes(t *testing.T, data *TestData, numPods int) (podN
 }
 
 func (data *TestData) testPodConnectivityDifferentNodes(t *testing.T) {
-	numPods := 2 // can be increased
+	numPods := clusterInfo.numNodes
```

Comment:
why? If we run on a large cluster, this can significantly increase the duration of the test, given that Pods are created sequentially.

Reply:
I only want to do this in hybrid mode where test both encap and noencap traffic. Will change